src/modules/schedule/components/calendar/calendar.tsx

```diff
@@ -64,11 +63,50 @@ export const Calendar: FC<CalendarProps> = () => {
           </div>
         )}
         allDaySlot={false}
-        // events={events}
         height={'75vh'}
-        buttonText={{ today: 'Today' }}
         nowIndicator
         slotMinTime={'07:00'}
+        events={events.map((event) => ({
+          id: event.id,
+          title: event.serviceProfessional.title,
+          start: set(event.date, {
+            hours: getHours(event.startTime),
+            minutes: getMinutes(event.startTime),
+            seconds: 0,
+            milliseconds: 0,
+          }),
+
+          end: set(event.date, {
+            hours: getHours(event.endTime),
+            minutes: getMinutes(event.endTime),
+            seconds: 0,
+            milliseconds: 0,
+          }),
+
+          className: styles.event,
+        }))}
+        eventContent={({ event }) => {
+          const startTime = format(
+            event.start ? new Date(event.start) : new Date(),
+            'HH:mm'
+          );
```

Comment:
I suggest you create helper `getTime` and then use it instead of writing const here
`{getTime(event.start)} - {getTime(event.end)}`